spock-specs/src/test/groovy/org/spockframework/docs/extension/InterceptorSpec.groovy

```diff
@@ -78,6 +93,16 @@ class InterceptorDemoExtension implements IAnnotationDrivenExtension<Interceptor
     featureInfo.addInitializerInterceptor new I('feature scoped initializer')
     featureInfo.addSetupInterceptor new I('feature scoped setup')
     featureInfo.addCleanupInterceptor new I('feature scoped cleanup')
+    // you call also perform a feature scoped interception of spec methods
```

Comment:
This sentence seams weird. 
Do you maybe mean:
"your call also performs a feature scoped interception of spec methods"

Reply:
s/call/can/